pubspec.yaml

```diff
@@ -33,6 +33,13 @@ dependencies:
   hive_flutter: ^1.0.0
   lint: ^1.0.0
   path_provider: ^2.0.1
+  hive: ^2.0.0
```

Comment:
'hive' and 'cupertino icon' packages are already declared. Remove the duplicate packages